aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -1,14 +1,15 @@
 ---
-title: .NET Generic Host
+title: ASP.NET Core generic host
```

Comment:
Have engineering put an :eye: on this. It was specifically named this way because this host applies to more than just ASP.NET.

Reply:
Then why we are putting the docs in the ASP.NET Core doc set?

Reply:
Fair. Because we're still working through the separation. They now have their own repo and build. It will make sense to separate the docs sooner than later.

Reply:
Got it.  Meanwhile I'll reverse my SEO branding. 😄